DXMainClient/Startup.cs

```diff
@@ -156,6 +157,25 @@ public void Execute()
             }
 #endif
 
+#if ISWINDOWS
+            if (UserINISettings.Instance.SteamIntegration)
+            {
+                try
+                {
+                    Logger.Log("Steam init called");
+#if YR || ARES
+                    SteamClient.Init(2229850);
+#else
+                    SteamClient.Init(2229880);
+#endif
+                }
+                catch (System.Exception e)
+                {
+                    Logger.Log("Steam init failed: " + e.Message);
+                    // Couldn't init for some reason (steam is closed etc)
+                }
+            }
+#endif
```

Comment:
Why this is Windows only? What about Linux?

Reply:
limited time to investigate *shipping* and calling the steamworks library in linux